### DISCUSSION TITLE
Source and Structure scale (0-1)

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#1018] [Research]
A few months ago I made this diagram to better understand the 'research process' or how different 'research meetings' might go, and I want to share it here as a general 'motivational scale' for projects, or meetings, or any other activities. 0 is more of an 'artistic' or 'uncertain' or 'unstructured', or 'source' motivation, and 1 is more 'structured' or 'logical' motivation, and before each meeting or whatever we are planning we can say, let's aim to be at .3 (more source), or .8 (more structure), and everyone has a sense of how the meeting (or space, or project) will unfold. 1 has pre-defined goals and objectives; the only 'goal' of 0 is to explore what is alive at the current time and see where it goes. Curious if anyone has any feedback :)

Just updated the picture*

![sourceandstructure](https://github.com/life-itself/community/assets/131390186/294cb2a5-3c60-4064-b7f5-dbf29008eeba)

**Comment by matthewmccarthy11** [comment]
It might also be an interesting way to consider the difference between different kinds of organizations: are 'teal' organizations, for example, more likely to hover around the .2 or .3 range? What does it mean to be motivated by source, how can that happen effectively? 

Important to note as well I don't think either is 'better' than the other and that a mix of the two is something to aim for. What does a good mix look like?

**Comment by matthewmccarthy11** [comment]
@asimong curious to get your initial thoughts here

**Comment by asimong** [comment]
For sure, thanks for the nudge @matthewmccarthy11 

First point, may be a little obvious but please bear with me if it is!
I don't see research, or any meaningful endeavour, as one, or the other, or at a certain point along that scale. Exhibit 1 I offer is the diamond of participatory decision-making: [on P2P wiki (my curation)](https://wiki.p2pfoundation.net/Diamond_of_Participation) or [according to Chris Corrigan](https://www.chriscorrigan.com/parkinglot/the-diamond-of-participation/), etc. It's always an organic, breath-like structure of expansive inspiration and contractive focusing down on what needs to be done now.

As seen organisationally, I see the two poles as, at one end, the archetypal network, with no aim of its own except to help people get together in the conceptual space stewarded by the network; and at the other end, work/task groups that set themselves up to do something, then dissolve when their task is done. 

Both are necessary. If there is nothing on the "source" end, then no community is built, people don't get to meet or trust each other to form the bonds needed for collaboration. Very common. If there is nothing on the "structure" end, then nothing significant actually gets done, and people drift off in search of places where they have more meaning and significance.

In terms of McGilchrist, the source corresponds roughly to the right hemisphere (and thus, left hand); the structure to the left hemisphere (the right hand that grasps and manipulates).

What I do think helps is if the community host can sense where the collective is, and make this aware to the participants. It is to be expected (as in the diamond) that there is first a divergent zone, then a groan zone, then a convergent zone. If everyone is roughly aligned on where in the "breath" we are, that helps greatly, because if they aren't there is likely to be talking across each other in terms of what "should" be happening; what we "should" be doing, etc.

Does that help?

And as I've hinted, I see the scale as different. A networking stage is naturally more open to more people, and inclusive (Rich Bartlett's "congregation"). A task stage is naturally fewer people (think Rich's "crew"), with sharper boundaries and exclusive to those who want to commit to collaboration on the chosen work area.

OK, there may be intermediate or mixed phases, but I'd say if that seems to be happening it needs to be managed with particular care.

**Reply by rufuspollock** [reply]
Inlining a couple of the images from the excellent referenced wiki page as i found them especially valuable:

![image](https://github.com/life-itself/community/assets/180658/8859e666-bb33-4c1f-b488-3446462ddd5e)

![image](https://github.com/life-itself/community/assets/180658/d80a4906-d53e-4223-ba5b-e24e6f2b600c)
